assets/wizards/advertising/views/services/index.js

```diff
@@ -68,6 +68,22 @@ const Services = ( { services, toggleService } ) => {
 				toggleOnChange={ value => toggleService( 'google_ad_manager', value ) }
 				titleLink={ google_ad_manager ? '#/google_ad_manager' : null }
 				href={ google_ad_manager && '#/google_ad_manager' }
+				notification={
```

Comment:
Suggestion: display the notice in the view with GAM ad units:

<img width="1116" alt="image" src="https://user-images.githubusercontent.com/7383192/133756050-50600947-46a2-4286-affe-a3b8c3c29c94.png">

Reply:
Good idea! Implemented at e056691eb5c51965d0da6d25be6ad0502a539f62 

I'm displaying it in both places so the user doesn't miss it in case they refresh or leave before accessing the Ad Units view.